content/index/index.html

```diff
@@ -12,25 +13,24 @@
 </head>
 <body>
   <main>
-    <h1>Index</h1>
-    <h2>About the Index</h2>
+    <h1>インデックス</h1>
+    <h2>インデックスについて</h2>
     <p>
-      This page includes a list of all ARIA design pattern examples indexed
-      either by role or by ARIA properties and states.
+      このページには、ARIA デザインパターンの例が、ロール別又は ARIA プロパティ及び状態別にインデックスされています。
     </p>
     <ul>
-      <li><a href="#examples_by_role_label">Examples by Role</a></li>
-      <li><a href="#examples_by_props_label">Examples by Properties and States</a></li>
-      <li><a href="#examples_experimental_label">Experimental Examples</a></li>
+      <li><a href="#examples_by_role_label">ロール別の例</a></li>
+      <li><a href="#examples_by_props_label">プロパティと状態別の例</a></li>
```

Comment:
「and」は「及び」がよいかもしれません。
```suggestion
      <li><a href="#examples_by_props_label">プロパティ及び状態別の例</a></li>
```